### DISCUSSION TITLE
Hide sensitive information in logs

### DISCUSSION THREAD

**Discussion by hrbust-liu** [apache/airflow#41948] [Q&A]
Airflow logs will turn admin and other information into *** Even if you configure hide_sensitive_var_conn_fields = False, it cannot be solved

airflow version is v2.6.3

**Comment by potiuk** [comment]
Change your passwords to not be "admin".  This is simply wrong to use "admin" as default password, never, ever do that. You've been warned.

You can also look at "advanced" logging in the docs and change default logging configuraiton to remove "secrets masker" filter if you really want

**Reply by hrbust-liu** [reply]
But my account and password do not contain the word "admin". Only my output log path contains

**Reply by potiuk** [reply]
Oh. I am quite sure you have "admin" set for a password somewhere. Look for it. Read how secrets masker work: https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/mask-sensitive-values.html - there is no hard-coded "admin"  masking - Airlfow will only mask whatever you have set as password. For example when you have "airflow" set as password, it will mask all "airflow".  And setting "airflow" as password is equally bad idea. Also "dupa" (often used in Polish) will be masked if you use it as password.

So as a general rule (and this is common sense) you should avoid setting your secrets as common words.
